### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
- Updated main README.
- Added/updated READMEs for individual apps to help users get up and running. 
- Added links to the Metabase store for the Premium option.
- Added gif to demo embed.
- Added links to other documentation on embedding Metabase.
- Changed the main prepare.sh script to download the latest Metabase version and run it in the repo's metabase folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,89 @@
...
 
-For the Django application
-`./manage.py runserver`
+This README will walk you through getting Metabase up and running, as well as a simple web app, to show you a live example of an embedded dashboard.
 
-For the Node application
-`node index.js`
+Here's a simple Metabase dashboard in action:
+
+![Simple Metabase dashboard embedded in Node application](/static/img/metabase_node_embed.gif)
```
Comment (dacort) [inline README.md:9]: WHAAAAAAT, nice gif!

File: README.md [hunk 2]
```diff
@@ -1,21 +1,89 @@
...
+
+## Premium embedding
+
+Embedding Metabase charts will always be free; we just ask you to include the "Powered by Metabase" graphic when embedding charts.
```
Comment (dacort) [inline README.md:13]:
```suggestion
Embedding Metabase charts will always be free, but we include a "Powered by Metabase" graphic when using the open source version.
```

File: README.md [hunk 3]
```diff
@@ -1,21 +1,89 @@
...
+
+If you'd like to remove the "Powered by Metabase" graphic, check out our [Premium Embedding](https://store.metabase.com/product/embedding) options.
```
Comment (dacort) [inline README.md:15]:
```suggestion
If you'd like to remove the "Powered by Metabase" attribution, check out our [Enterprise Edition](https://www.metabase.com/enterprise/) which lets you do so so as well as white-label the application.
```

File: README.md [hunk 4]
```diff
@@ -1,21 +1,89 @@
...
+
+## Prerequisites
+
+- Java version 8.x or 11.x. [Install OpenJDK](https://openjdk.java.net/install/).
+
+## Set up Metabase
+
+We'll first need to set up a running instance of Metabase to serve the embedded dashboards. If you already have an instance of Metabase running on your machine, shut it down.
```
Comment (dacort) [inline README.md:23]:
```suggestion
We'll first need to set up a running instance of Metabase to serve the embedded dashboards.
```

File: README.md [hunk 5]
```diff
@@ -1,21 +1,89 @@
...
+
+There are two ways to embed charts or dashboards in web applications:
+
+### Public embeds
+
+The Public embedding method is to simply use the public URLs inside of an iframe, or really anywhere you can insert HTML. The embedded dashboard has a secure URL, so a user can only look at the contents of the dashboard being shared. An end user never has information they can use to modify the URL and gain access to any other resources on your Metabase instance.
+
+### Signed embeds
+
+With signed embedding, all embedded charts and dashboards have to be signed using a secret key. You should sign embedded charts and dashboards on your server, which allows you to embed dashboards accessible to specific organizations, accounts, or users.
```
Comment (dacort) [inline README.md:63]:
```suggestion
With signed embedding, all embedded charts and dashboards have to be signed using a secret key. This allows you to build dynamic dashboards with a parameter that can be be locked down on the client side.  You should sign embedded charts and dashboards on your server, which allows you to embed dashboards accessible to specific organizations, accounts, or users.
```

File: README.md [hunk 6]
```diff
@@ -1,21 +1,89 @@
...
+
+## Documentation on embeds
+
+You can also learn more about embedding dashboards by reading [Metabase's documentation](https://www.metabase.com/docs/latest/).
+
+- [Quick search: embed](https://www.metabase.com/search.html?query=embed).
+
+Article links:
+
+- [Embedding Metabase in other applications](https://www.metabase.com/docs/latest/administration-guide/13-embedding.html)
```
Comment (dacort) [inline README.md:75]:
```suggestion
- [Embedding Metabase components in other applications](https://www.metabase.com/docs/latest/administration-guide/13-embedding.html)
```

Minor tweak to differentiate from the line below.

File: django/embedded_analytics/README.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
+# Django app
+
+This Django application demonstrates a simple, barebones Metabase dashboard embedded in a web application.
+
+## Prerequisites
+
+- **Metabase**. You should have already completed the setup detailed in the [README](../README.md) for this repository, which shows you how to get an instance of Metabase up and running in the [metabase](../metabase) directory of this repository.
```
Comment (dacort) [inline django/embedded_analytics/README.md:7]: Requires one more directory traversal or move this README one level up.

```suggestion
- **Metabase**. You should have already completed the setup detailed in the [README](../../README.md) for this repository, which shows you how to get an instance of Metabase up and running in the [metabase](../../metabase) directory of this repository.
```

File: django/embedded_analytics/README.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+- **Django**. You'll need to have [Django installed](https://docs.djangoproject.com/en/3.0/topics/install/) on your machine.
```
Comment (dacort) [inline django/embedded_analytics/README.md:9]: The `pip install` command below will install Django so we probably just need to tell folks they need Python installed.

File: laravel/embedded_analytics/README.md [hunk 9]
```diff
@@ -1,7 +1,52 @@
...
+# Laravel app
 
-1. Install dependencies: `composer install`. The package `lcobucci/jwt` has already been added
-2. See the Example Code in `app/routes/web.php`
-3. See the Example View in `resources/views/welcome.blade.php`
+This Laravel application demonstrates a simple, barebones Metabase dashboard embedded in a web application.
+
+## Prerequisites
+
+- **Metabase**. You should have already completed the setup detailed in the [README](../README.md) for this repository, which shows you how to get an instance of Metabase up and running in the [metabase](../metabase) directory of this repository.
```
Comment (dacort) [inline laravel/embedded_analytics/README.md:7]: Add another level of directory traversal here or move the README into the directory above.

File: django/embedded_analytics/README.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+
+3. Once the dependencies are installed, start the application by running: 
+
+```shell
+./manage.py runserver
+```
+
+4. Open your browser to [localhost:8000](http://localhost:8000).
+
+Explore the app to learn more about embedding Metabase charts and dashboards in applications. You can also check out the links to more documentation in the parent repository's main [README](../../README.md).
```
Comment (dacort) [inline django/embedded_analytics/README.md:29]: Update the README link if you choose to move the README one level up.

File: laravel/embedded_analytics/README.md [hunk 11]
```diff
@@ -1,7 +1,52 @@
...
+
+3. To start the application, run:
+
+```shell
+php artisan serve
+```
+
+4. Open your browser to [localhost:8000](http://localhost:8000) to see the app in action. 
+
+Explore the app to learn more about embedding Metabase charts and dashboards in applications. You can also check out the links to more documentation in the parent repository's main [README](../../README.md).
```
Comment (dacort) [inline laravel/embedded_analytics/README.md:45]: Update the README link if you choose to move the README one level up.

File: rails/embedded_analytics/README.md [hunk 12]
```diff
@@ -1,24 +1,30 @@
...
+This Rails application demonstrates a simple, barebones Metabase dashboard embedded in a web application.
 
-Things you may want to cover:
+## Prerequisites
 
-* Ruby version
+- **Metabase**. You should have already completed the setup detailed in the [README](../README.md) for this repository, which shows you how to get an instance of Metabase up and running in the [metabase](../metabase) directory of this repository.
 
-* System dependencies
+- **Rails**. You'll need to have [Ruby on Rails installed](https://guides.rubyonrails.org/getting_started.html) on your machine.
```
Comment (dacort) [inline rails/embedded_analytics/README.md:9]: Similar to Python, the `bundle install` will install Rails, so I think we just need to say they should have ruby and bundler installed.

File: rails/embedded_analytics/README.md [hunk 13]
```diff
@@ -1,24 +1,30 @@
...
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This Rails application demonstrates a simple, barebones Metabase dashboard embedded in a web application.
 
-Things you may want to cover:
+## Prerequisites
 
-* Ruby version
+- **Metabase**. You should have already completed the setup detailed in the [README](../README.md) for this repository, which shows you how to get an instance of Metabase up and running in the [metabase](../metabase) directory of this repository.
```
Comment (dacort) [inline rails/embedded_analytics/README.md:7]: Similar directory traversal fix - either add one more directory or move this README.

**Comment by dacort** [comment]
Tagging @mazameli in case he wants to take a look since there's a bunch of doc changes here.

File: README.md [hunk 14]
```diff
@@ -1,21 +1,87 @@
-# embedding-reference-apps
-Reference applications for common web frameworks showing how to embed Metabase charts
+# Embedding Metabase dashboards: reference web apps
 
-# Running the apps
-1. Run the metabase server with `java -jar metabase.jar` from the `/metabase` directory
-2. Run the relevant reference application
+This repo contains example applications for common web frameworks to demonstrate how [Metabase](https://www.metabase.com/) dashboards embed in web applications.
```
Comment (kdoh) [inline README.md:3]:
```suggestion
This repo contains example applications for common web frameworks to demonstrate how [Metabase](https://www.metabase.com/) dashboards can be embedded in your application.
```

**Pull request merged**